blockchain/pallets/ddcpayouts.go

```diff
@@ -0,0 +1,213 @@
+package pallets
+
+import (
+	"errors"
+	"reflect"
+
+	gsrpc "github.com/centrifuge/go-substrate-rpc-client/v4"
+	"github.com/centrifuge/go-substrate-rpc-client/v4/scale"
+	"github.com/centrifuge/go-substrate-rpc-client/v4/types"
+	"github.com/centrifuge/go-substrate-rpc-client/v4/types/codec"
+)
+
+const (
+	MaxBatchesCount = 1000
+	MaxBatchSize    = 1000
+)
+
+type BatchIndex = types.U16
+
+type BillingReport struct {
+	State                     State
+	Vault                     types.AccountID
+	TotalCustomerCharges      CustomerCharge
+	TotalDistributedReward    types.U128
+	TotalNodeUsage            NodeUsage
+	ChargingMaxBatchIndex     BatchIndex
+	ChargingProcessedBatches  []BatchIndex
+	RewardingMaxBatchIndex    BatchIndex
+	RewardingProcessedBatches []BatchIndex
+}
+
+type BillingReportDebt struct {
```

Comment:
is it used anywhere?

Reply:
No, removed as well as the other unused definitions.